frontend/integration-tests/tests/operator-hub/operator-hub.scenario.ts

```diff
@@ -1,19 +1,38 @@
-import { browser, $, ExpectedConditions as until, element, by } from 'protractor';
+import { browser, ExpectedConditions as until } from 'protractor';
 import { execSync } from 'child_process';
 
 import { appHost, checkLogs, checkErrors, testName } from '../../protractor.conf';
 import * as crudView from '../../views/crud.view';
 import * as catalogView from '../../views/catalog.view';
 import * as catalogPageView from '../../views/catalog-page.view';
-import * as operatorHubView from '../../views/operator-hub.view';
-
-describe('Subscribing to an Operator from OperatorHub', () => {
-  let installedOperator: string;
+import * as sidenavView from '../../views/sidenav.view';
+
+describe('Interacting with OperatorHub', () => {
```

Comment:
This scenario now just tests the tile view functionality (filtering, sorting, etc). Actual Operator installation are found in the other two tests.